app/controllers/v1/hackbot/webhook_controller.rb

```diff
@@ -8,8 +8,10 @@ def callback
         elsif params[:type] != 'event_callback'
           render status: :not_implemented
         else
-          handle_message
           render status: 200
+
```

Comment:
I'm unsure here how to achieve the of threads without using them.

Effectively, I want the HTTP request to closed so that Slack stops sending us duplicate messages.

idk, something like:

```
render status: 200

close

handle_message
```